pkg/core/typeurl/types_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package typeurl is
+package typeurl
+
+import (
+	"testing"
+)
+
+func TestMarshalAny(t *testing.T) {
```

Comment:
perhaps later we can change to table-driven tests ? see this https://github.com/kubeedge/kubeedge/blob/master/cloud/test/integration/crds/validation_test.go for reference.

Reply:
thank you